src/foundry/foundry.js/audioHelper.d.ts

```diff
@@ -108,130 +114,81 @@ declare class AudioHelper {
    */
   static hasAudioExtension(src: string): boolean;
 
-  /* -------------------------------------------- */
-
   /**
-   * Play a single audio effect by it's source path and Howl ID
+   * Given an input file path, determine a default name for the sound based on the filename
+   * @param src - An input file path
+   * @returns A default sound name for the path
    */
-  play(src: string, id: number): void;
+  static getDefaultSoundName(src: string): string;
 
-  /* -------------------------------------------- */
+  /**
+   * Play a single Sound by providing its source.
+   * @param src     - The file path to the audio source being played
+   * @param options - Additional options passed to Sound#play
+   * @returns The created Sound which is now playing
+   */
+  play(src: string, options?: Sound.PlayOptions): Promise<Sound>;
 
   /**
-   * Register an event listener to await the first mousemove gesture and begin
-   * playback once observed
+   * Register an event listener to await the first mousemove gesture and begin playback once observed
    */
   awaitFirstGesture(): void;
 
-  /* -------------------------------------------- */
-
   /**
-   * Handle the first observed user gesture
-   * @param event - The mouse-move event which enables playback
+   * Request that other connected clients begin preloading a certain sound path.
+   * @param src - The source file path requested for preload
+   * @returns A Promise which resolves once the preload is complete
    */
-  protected _onFirstGesture(event: Event): void;
-
-  /* -------------------------------------------- */
-
-  preload(data: any): void;
-
-  /* -------------------------------------------- */
-  /*  Socket Listeners and Handlers               */
-  /* -------------------------------------------- */
+  preload(src: string): Promise<Sound>;
 
   /**
    * Open socket listeners which transact ChatMessage data
+   * @internal
```

Comment:
Why internal? Shouldn't this be `protected` instead?

> and _ naming and @protected both mean we annotate the thing with protected

Reply:
That's what @cswendrowski told me is the right thing: https://discord.com/channels/170995199584108546/811676497965613117/873052279589208064
> `AudioHelper#_activateSocketListeners` is probably private along with all other `activateSocketListener` methods, we'll likely mark it `@private` but remove the `_`